docs/src/getting-started/beginner-guide.rst

```diff
@@ -0,0 +1,279 @@
+.. _beginner_guide:
+
+Beginner's Guide to Machine Learning for Atomistic Systems
+==========================================================
```

Comment:
Keeping this up to date will be a huge pain for us. If we keep this page I would rather keep a very short list maybe of the terms that we use throughout the docs and link to a better intros like for example for in the scientific literature

Unke, O. T. et al. “Machine Learning Force Fields”,  10.1021/acs.chemrev.0c01111

and

Andrew D. White, Deep learning for molecules & materials,  https://dmol.pub/

or more pratical our aml-intro repo which has lectures and examples

https://github.com/ceriottm/aml-intro 

or the nice GNN for chemist repo, which also has lectures and examples:

https://github.com/HFooladi/GNNs-For-Chemists